### ISSUE TITLE
Extra prop minimumNights passed to DayPicker

### ISSUE THREAD

**Issue by joux3** [airbnb/react-dates#685]
This seems to have been introduced in 12.5.0. Full trace:
```
warning.js:35 Warning: Failed prop type: DayPicker: unknown props found: minimumNights
    in DayPicker (created by DayPickerSingleDateController)
    in DayPickerSingleDateController (created by SingleDatePicker)
    in div (created by SingleDatePicker)
    in div (created by OutsideClickHandler)
    in OutsideClickHandler (created by SingleDatePicker)
    in div (created by SingleDatePicker)
    in SingleDatePicker (created by SingleDatePickerWrapper)
    in SingleDatePickerWrapper
    in div (created by Story)
    in div (created by Story)
    in Story
    in div
    in div
```

Repro: `npm run storybook`, select `SingleDatePicker (SDP) -> default`, click the date input and observe the failed prop validation in console.

**Comment by toddkblake** [comment]
+1

**Comment by ljharb** [comment]
Fixed in #686
